doc/UsersGuide/source/BuildingRunningTesting/HAFSCodeContributionGuidelines.rst

```diff
@@ -0,0 +1,82 @@
+.. _HAFSCodeContributionGuidelines:
+
+***********************************************
+UFS Repository Code Management Guidance (Draft)
+***********************************************
+
+
+Repository
+----------
```

Comment:
Normally, the [order of headings](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections) in Sphinx means that `===` comes after `***`. 
```suggestion
==========
```
This should be changed throughout this chapter.